dashboard/app/models/lesson_activity.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: lesson_activities
+#
+#  id         :integer          not null, primary key
+#  lesson_id  :integer          not null
+#  position   :integer          not null
+#  properties :string(255)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_lesson_activities_on_lesson_id  (lesson_id)
+#
+
+class LessonActivity < ApplicationRecord
```

Comment:
This was the best name I could come up with. PRO/CON:
* PRO: accurate and specific
* CON: sounds like a join table name. could someday lead to a join table with a confusing name like `lesson_activity_lessons`, which would happen if we allowed activities to exist in multiple lessons. our current plan is for each activity (and most other data types other than Level) to only have one parent in the hierarchy, so I am not to worried about this.

In the future, if the `Activity` class is removed, we could rename this model to `Activity`. there are relatively few references to the model. We probably won't ever remove/rename the `activities` table though, because there are likely many data analysis scripts which depend on that name.

Other name ideas for this model / table are welcome!

Reply:
I think this name makes the most sense to me.

Reply:
I had a realization which is that you can tell from the singular_plural pattern that this is not a join. a join model / table would be named LessonsActivity / lessons_activities, not LessonActivity / lesson_activities. it seems like a small difference but it makes me feel much better about this name. it is only one letter different but it makes this name sound right to me.